namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlService.scala

```diff
@@ -410,7 +410,7 @@ class HttpControlService(storage: DtabStore, delegate: Ns => NameInterpreter, na
   private[this] val enumeratingNamers = namers.values.collect {
     case namer: EnumeratingNamer => namer
   }.toSeq
-  private[this] val boundNames =
+  private[this] def boundNames =
     Activity.collect(enumeratingNamers.map(_.getAllNames))
       .map(_.toSet.flatten)
```

Comment:
Similarly, is this necessary to be a def?  I'd expect it be to be fine/good for this to be static (potentially a lazy val?).